docs/tutorials/mobile/index.md

```diff
@@ -18,21 +18,47 @@ ONNX Runtime gives you a variety of options to add machine learning to your mobi
 
 ![Steps to build for mobile platforms](../../../images/mobile.png){:width="60%"}
 
-1. Which ONNX Runtime mobile library should I use?
+1. Which ONNX Runtime package should I use?
 
-   We publish the following ONNX Runtime mobile libraries:
-   * Android C/C++
-   * Android Java
+   We publish the following ONNX Runtime packages that can be used in mobile applications:
+   * Android Java/C/C++
+     * Mobile (onnxruntime-mobile) and full (onnxruntime-android) packages
    * iOS C/C++
-   * iOS Objective C
+     * Mobile (onnxruntime-mobile-c) and full (onnxruntime-c) packages
+   * iOS Objective-C
+     * Mobile (onnxruntime-mobile-objc) and full (onnxruntime-objc) packages
+
+   The full package has the full ONNX Runtime feature set.
+   The mobile package has a smaller binary size but limited feature support, like a reduced set of operator implementations and no support for running ONNX models.
+
+   A [custom build](../../build/custom.md) is tailored to your model(s) and can be even smaller than the mobile package. However, using a custom build is more involved than using one of the published packages.
+
+   If the binary size of the full package is acceptable, using the full package is recommended because it is easier to use.
+   Otherwise, consider using the mobile package or a custom build.
+
+   To give an idea of the binary size difference between mobile and full packages:
+
+   ONNX Runtime 1.11.0 Android package `jni/arm64-v8a/libonnxruntime.so` file size:
+   |Package|Size|
+   |-|-|
+   |onnxruntime-mobile|3.3 MB|
+   |onnxruntime-android|12 MB|
```

Comment:
That's a larger size than expected. Wasn't the mobile package around 2.5MB a couple of releases ago? 

We did add some more opsets and runtime optimization I guess, but I wasn't expecting it to be > 3MB.

Reply:
checked the packages here:
[https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime-mobile](https://search.maven.org/artifact/com.microsoft.onnxruntime/onnxruntime-mobile)

1.10.0: 2.9 MB
1.9.0: 2.9 MB
1.8.2: 2.7 MB

Reply:
I expected about 150kb from runtime optimization and QDQ support so a 400kb increase is surprising. There are some additional MLAS kernels but usually those are small. Definitely something to look into.